CondFormats/JetMETObjects/interface/METCorrectorParameters.h

```diff
@@ -23,22 +23,23 @@ class METCorrectorParameters
       public:
         //-------- Constructors -------------- 
         Definitions() {}
-        Definitions(const std::vector<std::string>& fVar, const std::vector<std::string>& fParVar, const std::string& fFormula); 
+        Definitions(const std::vector<std::string>& fVar, const std::vector<int>& fParVar, const std::string& fFormula); 
         Definitions(const std::string& fLine); 
         //-------- Member functions ----------
         unsigned nBinVar()                  const {return mBinVar.size(); }
         unsigned nParVar()                  const {return mParVar.size(); }
-        std::vector<std::string> parVar()   const {return mParVar;        }
+        int ptclType()                      const {return ptclType_; }
+        std::vector<int> parVar()   const {return mParVar;        }
         std::vector<std::string> binVar()   const {return mBinVar;        } 
-        std::string parVar(unsigned fIndex) const {return mParVar[fIndex];}
+        int parVar(unsigned fIndex) const {return mParVar[fIndex];}
         std::string binVar(unsigned fIndex) const {return mBinVar[fIndex];} 
         std::string formula()               const {return mFormula;       }
       private:
         //-------- Member variables ----------
-	int 			ptclType;
+	int 			 ptclType_;
```

Comment:
@davidlange6 
it is definitely a schema change. In principle we should not allow this. But after a check, there are no payloads from this class stored in the DB. Based on this I gave my ok to the PR. However, any accidental use of a previous release to create payloads, will generate unreadable data.